spec/forms/claimant_form_spec.rb

```diff
@@ -68,6 +68,20 @@
       end
     end
 
+    describe 'other_title' do
+      before { claimant_form.title = 'Other' }
```

Comment:
You are setting the title outside of the context named 'when title is other' which doesn't read so well.